docker/Dockerfile

```diff
@@ -15,7 +16,7 @@ RUN yum install -y make git gcc libcurl-devel sqlite-devel && \
     && gosu nobody true
 RUN mkdir -p /onedrive/conf /onedrive/data
 ADD onedrive /usr/src/onedrive
-RUN . /root/dlang/dmd-2.083.0/activate && \
+RUN . `bash install.sh -a` && \
```

Comment:
It looks like dmd installation is flaky. I ran ```docker build . --no-cache``` and it showed me problem wit pgp key again...
```gpg: no valid OpenPGP data found.```

Can we rely on this? Maybe it would be better to setup local cache/clone for this files?

Reply:
Then again it could be my ISPs failt, we can be only sure once we have docker builds on board, as requested in https://github.com/abraunegg/onedrive/issues/281